### PR TITLE
Updating the gui documentation.

### PR DESCRIPTION
**Pull request by gautam1168**
Update the gui.rst file to provide instructions for running the GUI

### CODE REVIEW

**Comment by gautam1168** [comment]
@wkerzendorf Here is the PR. Should I add some screenshots to show all the features of the GUI. Some of the features are not obvious to first time users. Like the double click to see ion and level populations from the shell table.

File: docs/gui.rst [hunk 1]
```diff
@@ -6,12 +6,39 @@ TARDIS uses the `PyQt4 framework <http://www.riverbankcomputing.com/software/pyq
...
-GUI Layout and Features
------------------------
+gui.py contains all the classes used to create the GUI for Tardis.
+
+This module must be imported inside IPython console started above. The console provides the event loop and the place
+to create/calculate the tardis model. So the module is basically a tool to visualize results. 
 
+Running instructions
+--------------------
+    1. Decide which Qt binding you want to use (PySide or PyQt) and
```
Comment (wkerzendorf) [inline docs/gui.rst]: So a good way to write this is

```
Decide which Qt binding you want to use (PySide or PyQt) and accordingly set QT_API in shell::
   export QT_API=pyside
or::

   export QT_API=pyqt
```

**Comment by wkerzendorf** [comment]
Sounds good

**Comment by gautam1168** [comment]
@wkerzendorf I have added the snapshots and updated the instructions in the running.rst as well. Please take a look.

File: tardis/gui.py [hunk 2]
```diff
@@ -1,11 +1,5 @@
 """gui.py contains all the classes used to create the GUI for Tardis.
 
-3. routine listings
```
Comment (gautam1168) [inline tardis/gui.py]: This was left here by mistake.

**Comment by wkerzendorf** [comment]
@gautam1168 Have you made the documentation and looked at it in HTML? If you go to the docs directory and do `make html` and then open `_build/html/index.html` you can see the documentation. I think it's close but not quite there yet. Maybe we can chat next week.

**Comment by gautam1168** [comment]
Yes I did check the HTML. But okay if you want some changes. Also I made
the sub package. Its the other PR. Did you see that?

**Comment by wkerzendorf** [comment]
@gautam1168 yes, I did. Haven't commented though (it's just 11am here, I think now the timezones are really problematic :wink: )

File: docs/gui.rst [hunk 3]
```diff
@@ -6,12 +6,67 @@ TARDIS uses the `PyQt4 framework <http://www.riverbankcomputing.com/software/pyq
...
 IPython. The above command to start IPython accomplishes this.
 
+gui.py contains all the classes used to create the GUI for Tardis.
+
+This module must be imported inside IPython console started above. The console provides the event loop and the place
+to create/calculate the tardis model. So the module is basically a tool to visualize results. 
+
+Running instructions
+--------------------
+    Decide which Qt binding you want to use (PySide or PyQt) and
```
Comment (wkerzendorf) [inline docs/gui.rst]: no indent

File: docs/gui.rst [hunk 4]
```diff
@@ -6,12 +6,67 @@ TARDIS uses the `PyQt4 framework <http://www.riverbankcomputing.com/software/pyq
...
+
+This module must be imported inside IPython console started above. The console provides the event loop and the place
+to create/calculate the tardis model. So the module is basically a tool to visualize results. 
+
+Running instructions
+--------------------
+    Decide which Qt binding you want to use (PySide or PyQt) and 
+    accordingly set QT_API in shell::
+
+            export QT_API=pyside
```
Comment (wkerzendorf) [inline docs/gui.rst]: just one indent

File: docs/gui.rst [hunk 5]
```diff
@@ -6,12 +6,67 @@ TARDIS uses the `PyQt4 framework <http://www.riverbankcomputing.com/software/pyq
...
+            or:
+            export QT_API=pyqt
+            
+    Start the IPython console with eventloop integration::
```
Comment (wkerzendorf) [inline docs/gui.rst]: no indent

File: docs/gui.rst [hunk 6]
```diff
@@ -6,12 +6,67 @@ TARDIS uses the `PyQt4 framework <http://www.riverbankcomputing.com/software/pyq
...
+    
+            ipython --pylab=qt4
+            
+    Display your model
```
Comment (wkerzendorf) [inline docs/gui.rst]: no indent and double colons: `Display your model::`

File: docs/gui.rst [hunk 7]
```diff
@@ -6,12 +6,67 @@ TARDIS uses the `PyQt4 framework <http://www.riverbankcomputing.com/software/pyq
...
+    
+    .. code-block:: python
+    
+        from tardis import gui 
+        win = gui.Tardis()
+        win.show_model(mdl)
+
+Exceptions
```
Comment (wkerzendorf) [inline docs/gui.rst]: that's not really necessary.

**Comment by gautam1168** [comment]
@wkerzendorf  I made the changes. But I don't know where I should commit them. Right now the branches are like:
gui-documentation-update-----unrelated-commit1------unrelated-commit2-------unrelated-commit3-------gui-package-----unstaged-changes

So should I just checkout to gui-documentation-update, stage and commit my changes to that place and create a new branch like:
                                  ___new-commit-with-corrections-----gui-documentation-update
                                /       
last-related-commit-----unrelated-commit1------unrelated-commit2-------unrelated-commit3-------gui-package

Or is there something else?

**Pull request merged**